src/lib/services/api-token-service.ts

```diff
@@ -193,6 +213,7 @@ export class ApiTokenService {
         secret: string,
     ): Promise<IApiUser | undefined> {
         const token = await this.getTokenWithCache(secret);
+        this.logger.info(`getUserForToken ${secret} found: ${token}`);
```

Comment:
That's probably gotta come out so just a reminder that we gotta revert this one when we're done

Reply:
absolutely